### PR TITLE
Migration: alert-list file

### PR DESCRIPTION
**Pull request by Chanputhi**
_Any italic text should be deleted from the final Pull Request text, including this line_

# Description

_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change._

Fixes # (issue)

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

## Testing Checklist:

- [ ] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from @macite and @jakerenzella on the Pull Request

### CODE REVIEW

**Comment by Chanputhi** [comment]
I have implemented this code and it should be correct already. However, this component has dependency and waiting for the alertService service to make it work.

File: src/app/common/alert-list/alert-list.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AlertListComponent } from './alert-list.component';
+
+describe('NotFoundComponent', () => {
```
Comment (lst97) [inline src/app/common/alert-list/alert-list.component.spec.ts]: This is my sample code, the name should not be NotFoundComponent, please check

**Review by lst97 — CHANGES_REQUESTED** [review]

File: src/app/common/alert-list/alert-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Component, Inject } from "@angular/core";
+import { alertService } from "src/app/ajs-upgraded-providers";
+
+@Component({
+    selector: "alert-list",
+    templateUrl: "./alert-list.component.html",
+})
+export class AlertListComponent {
+    constructor(@Inject(alertService) public alertService : any) {
+
```
Comment (lst97) [inline src/app/common/alert-list/alert-list.component.ts:10]: It is not satisfy, because no logic implemented here, please check.

**Review by lst97 — CHANGES_REQUESTED** [review]

**Comment by lst97** [comment]
Please convert it do draft

File: src/app/common/alert-list/alert-list.component.spec.ts (hunk 1 above)
Comment (Chanputhi) [reply src/app/common/alert-list/alert-list.component.spec.ts]: Yes, I have rename it now.

**Pull request closed without merging**